go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	cloud.google.com/go/storage v1.22.1
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.19.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.11.0
-	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78
+	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1
```

Comment:
Also a big jump here (similar to https://github.com/gravitational/teleport/pull/16314), plus some reasonable jumps in other indirect dependencies. Should we do any additional testing to make sure it's alright?